### PR TITLE
Do not display smart app banner on the core profiler, LYS, and CYS pages

### PR DESCRIPTION
**Pull request by moon0326**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

This PR removes smart app banner from the core profiler, LYS, and CYS pages.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Use your iPhone or Android.
3. Open the core profiler, LYS, and CYS pages.
4. Confirm no app banner displayed.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

Do not display smart app banner on the core profiler, LYS, and CYS pages

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/src/Internal/Admin/Loader.php [hunk 1]
```diff
@@ -319,7 +329,13 @@ public static function add_component_settings( $settings ) {
 			);
 		}
 
+		/**
+		 * The woocommerce_component_settings_preload_endpoints filter
+		 *
+		 * @since 6.5.0
```
Comment (moon0326) [inline plugins/woocommerce/src/Internal/Admin/Loader.php:335]: Using 6.5.0 since that's the first version released after we merged WC Admin into the core.

**Review by chihsuan — APPROVED** [review]
Nice. LGTM and tested well!  🚢

File: plugins/woocommerce/src/Internal/Admin/Loader.php [hunk 2]
```diff
@@ -189,7 +190,16 @@ public static function add_admin_body_classes( $admin_body_class = '' ) {
 	 * See https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html
 	 */
 	public static function smart_app_banner() {
-		if ( PageController::is_admin_or_embed_page() ) {
+		$exlude_paths = array(
```
Comment (ilyasfoo) [inline plugins/woocommerce/src/Internal/Admin/Loader.php]: Nit: typo

```suggestion
		$exclude_paths = array(
```

**Review by ilyasfoo — APPROVED** [review]
Thanks for fixing, @moon0326! While testing CYS, the app banner persisted from homescreen, but I think that's expected since it's a react navigation. I wonder if there is a way to remove the banner after it's shown? But I digress, it might be worse UX anyway.

I have a nit comment, pre-approving! 🚢

**Pull request merged**